external/unbound/dnscrypt/dnscrypt_config.h

```diff
@@ -0,0 +1,17 @@
+#ifndef UNBOUND_DNSCRYPT_CONFIG_H
+#define UNBOUND_DNSCRYPT_CONFIG_H
+
+/*
+ * Process this file (dnscrypt_config.h.in) with AC_CONFIG_FILES to generate
+ * dnscrypt_config.h.
+ *
+ * This file exists so that USE_DNSCRYPT can be used without including config.h.
+ */
+
+#if 0 /* ENABLE_DNSCRYPT */
+# ifndef USE_DNSCRYPT
+#  define USE_DNSCRYPT 1
+# endif
+#endif
+
+#endif /* UNBOUND_DNSCRYPT_CONFIG_H */
```

Comment:
Note: this file is **not** in the latest release but is instead a hacked `dnscrypt_config.h.in`. In the future, I think we should aim to adapt our build process to work cleanly with other projects (within reason). This includes working from an unbound submodule with monero-specific branch so svn rebasing can be applied cleanly while keeping the log intact for review.

Referencing 
https://github.com/anonimal/unbound/commit/a48c8c5ba0e8d97b0c00950a5f49d9f2cf5ffa39